src/table/settings/pathfinding_settings.ini

```diff
@@ -49,33 +49,6 @@ var      = pf.roadveh_queue
 def      = true
 cat      = SC_EXPERT
 
-[SDT_BOOL]
-var      = pf.new_pathfinding_all
-to       = SLV_87
-def      = false
-cat      = SC_EXPERT
-
-[SDT_BOOL]
-var      = pf.yapf.ship_use_yapf
-from     = SLV_28
-to       = SLV_87
-def      = false
-cat      = SC_EXPERT
-
-[SDT_BOOL]
-var      = pf.yapf.road_use_yapf
-from     = SLV_28
-to       = SLV_87
-def      = true
-cat      = SC_EXPERT
-
-[SDT_BOOL]
-var      = pf.yapf.rail_use_yapf
-from     = SLV_28
-to       = SLV_87
-def      = true
-cat      = SC_EXPERT
```

Comment:
Why did you remove these? If I read it right, they are just to convert old settings into new settings .. but both PFs still exist. So I wouldn't exist these to be removed?

Reply:
Mostly laziness - they're extremely old compatibility settings only (removed in 2008-era), and they were getting in the way of removing the `YapfSettings` data structure. Saves still relying on it will "just" get reset to YAPF.

Sure, technically a regression, but not a significant one IMO

Reply:
We take so much effort in keeping old savegames as compatible as we can .. I don't feel this is the right call. It is a very small effort on our part to fix this :)

Reply:
I've restored the SL for the variable as well, I think that was the right thing to do?